server/src/main/java/org/spine3/server/commandbus/CommandBus.java

```diff
@@ -445,7 +462,11 @@ private Builder() {
 
         /**
          * Builds an instance of {@link CommandBus}.
+         *
+         * <p>This method is supposed to be called internally when building an aggregating
```

Comment:
I would say, `...enclosing {@code BoundedContext}.` to avoid confusion with aggregates.